.github/workflows/build_icons.yml

```diff
@@ -33,10 +30,12 @@ jobs:
         with:
           name: geckodriver-log
           path: ./geckodriver.log
-      - name: Running gulp default task for building devicon.min.css
-        run: |
-          gulp default
-      - name: Commit changes
-        uses: stefanzweifel/git-auto-commit-action@v4
+      - name: Running npm task for building devicon.min.css
+        run: npm run build-css
+      - name: Create new branch and push it
+        uses: technote-space/create-pr-action@v2
         with:
-          commit_message: Build new icons, icomoon.json and devicon.css
+          COMMIT_MESSAGE: "Built new icons, icomoon.json and devicon.css"
+          PR_BRANCH_PREFIX: "build/feature/"
+          PR_BRANCH_NAME: "test-123"
```

Comment:
Maybe this could a little more dynamic? ;-)

Reply:
Haha, sorry about that. I was in the middle of testing the trigger and wasn't sure how to do string interpolations in yml files.